### PR TITLE
fix msimonin/ombt-orchestrator#57

### PR DESCRIPTION
**Pull request by jrbalderrama**

### CODE REVIEW

File: orchestrator/campaign.py [hunk 1]
```diff
@@ -371,19 +392,32 @@ def incremental_campaign(test, provider, force, pause, config, env):
...
                 dump_parameters(env_dir, current_parameters)
+                t.reset(env=env_dir)
                 time.sleep(pause)
             sweeper.done(current_group)
         except (AttributeError, EnosError, RuntimeError, ValueError, KeyError, OSError) as error:
             sweeper.skip(current_group)
-            print(error, file=sys.stderr)
-            print(error.args, file=sys.stderr)
+            import traceback
+            traceback.print_exc()
```
Comment (msimonin) [inline orchestrator/campaign.py]: to be removed :)

Comment (jrbalderrama) [reply orchestrator/campaign.py]: fix with https://github.com/msimonin/ombt-orchestrator/pull/123/commits/7bb369d625b97cc02f12fdcbb81f097009d973c8

File: orchestrator/campaign.py [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
 
-def filter_1(parameters):
-    return filter_params(parameters, condition=lambda x: x['nbr_servers'] <= x['nbr_clients'])
+def filter_1(condition, parameters):
+    if not condition:
+        condition = (lambda x: x['nbr_servers'] <= x['nbr_clients'])
+    return filter_params(parameters, condition=condition)
 
 
 def filter_2(parameters):
```
Comment (msimonin) [inline orchestrator/campaign.py]: Should we add `condition` ?

Comment (jrbalderrama) [reply orchestrator/campaign.py]: done !

**Pull request merged**